tests/scripts/task_build.sh

```diff
@@ -16,4 +16,4 @@
 # specific language governing permissions and limitations
 # under the License.
 export VTA_HW_PATH=`pwd`/3rdparty/vta-hw
-cd $1 && cmake .. && make $2 && cd ..
+cd $1 && cmake .. -DCMAKE_BUILD_TYPE=Debug && make $2 && cd ..
```

Comment:
Is this change intentional? Perhaps it makes sense to test with `RelWithDebInfo`?

Reply:
This change is intensional. With this change, we get clearer backtraces from CI.

Reply:
Yeah i think it makes sense to have a clear backtrace in CI, but it also occurs to me that some bugs may not always appear in Debug mode, for example, mistakenly returning a reference to a stack variables. Given that we use Release mode in production, it is good that the CI aligns well.

Reply:
done